internal/process/defunct_processes.go

```diff
@@ -1,59 +1,90 @@
 package process
 
 import (
-	"fmt"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 	"os"
 	"strconv"
 	"strings"
 )
 
-// DefunctProcesses returns the PIDs of all the zombie processes in the node
-func DefunctProcesses() ([]int, error) {
-	directories, err := os.Open("/proc")
+// ProcessFS is the process file system.
+const ProcessFS = "/proc"
+
+// Stat represents status information of a process from /proc/[pid]/stat.
+type Stat struct {
+	// Fname is the filename of the executable.
+	Fname string
+
+	// State is the state of the process.
+	State string
+}
```

Comment:
This struct allows `status()` to return the executable name of the process (along with its state) which is needed for logging as mentioned in https://github.com/cri-o/cri-o/pull/5082#discussion_r672302364.